### ISSUE TITLE
How would I configure to create a pause before starting playback when spotify is first started?

### ISSUE THREAD

**Issue by giggywithit** [dtcooper/raspotify#374]
Hello

I have a system that requires 1 second at startup for the amplifier and would like to configure if possible a slight delay for when the audio device is first opened. Not between tracks or anything, just a pause in playback when the system goes from being unused to an active state. Any pointers? All I need is like a second or 1.5 seconds delay at start.

Thx

**Comment by JasonLG1979** [comment]
duplicate of https://github.com/dtcooper/raspotify/issues/93
